p2p/net/udp.go

```diff
@@ -81,14 +90,76 @@ func newUDPListener(listenAddress *net.UDPAddr) (*net.UDPConn, error) {
 
 var IPv4LoopbackAddress = net.IP{127, 0, 0, 1}
 
+func (n *UDPNet) initSession(to string, remote p2pcrypto.PublicKey) (NetworkSession, error) {
+	raddr, err := resolveUDPAddr(to)
+	if err != nil {
+		return nil, err
+	}
+
+	session := createSession(n.local.PrivateKey(), remote)
+	handshakeMessage, err := generateHandshakeMessage(session, n.config.NetworkID, 1010, n.local.PublicKey())
+	if err != nil {
+		return nil, err
+	}
+	_, err = n.conn.WriteToUDP(handshakeMessage, raddr)
+
+	return session, nil
+}
+
+func (n *UDPNet) handleSession(message []byte) (NetworkSession, error) {
```

Comment:
`handleSession` doesn't say much on what the purpose of this method and when it should be called. Rename it or add documentation